addons/mail/models/mail_thread.py

```diff
@@ -4699,6 +4699,11 @@ def _thread_to_store(self, store: Store, fields, *, request_list=None):
                 res["suggestedRecipients"] = thread._message_get_suggested_recipients(
                     reply_discussion=True, no_create=True,
                 )
+            if request_list and "contact_fields" in request_list:
```

Comment:
Could be great to have a ``request_list = request_list or []`` at method beginning so that you could simplify all those "if request_list and [something] in request_list``

Reply:
Great idea!